traveler-swift-core/TravelerKit/Extensions/Timezone+UTCOffset.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Timezone+UTCOffset.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-07-11.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+extension TimeZone {
+    init? (with utcOffsetHours:String) {
```

Comment:
spacing

Reply:
String? I think this should take in a different type...offset minutes or offset seconds or offset milliseconds.
whatever is most common